### PR TITLE
Apply image scale operations to multiple cells

### PR DESCRIPTION
**Pull request by t2b3**
Loop through all cells in selection and apply the
scale operation.

### CODE REVIEW

File: src/document.h [hunk 1]
```diff
@@ -84,6 +84,7 @@ struct Document {
     int editfilter;
 
     Vector<Cell *> itercells;
+    Vector<Cell *> queuedcells;
```
Comment (aardappel) [inline src/document.h:87]: There's no reason to make this part of the document

File: src/document.h [hunk 2]
```diff
@@ -1500,6 +1500,41 @@ struct Document {
...
+                            c->text.image->BitmapScale(sc);
+                        } else {
+                            c->text.image->DisplayScale(sc);
+                        }
+                        c->ResetLayout();
+                    }
+                    Refresh();    
+                    return nullptr;
+                } else {
+                    return _(L"No image found in selection.");
```
[the remark this answers is not in the document]
Comment (aardappel) [reply src/document.h:1528]: can you try this approach? there's no need to loop twice or store anything

[the remark this answers is not in the document]
Comment (t2b3) [reply src/document.h]: Thanks for your review! I force-pushed the branch. It should look much better by now.

**Comment by aardappel** [comment]
Thanks!

**Pull request merged**